### DISCUSSION TITLE
Live notifications no longer notify unless you have the chat open?

### DISCUSSION THREAD

**Discussion by Aerglonus** [Chatterino/chatterino2#3739] [Q&A] [answered]
I recently noticed I wasn't getting live notification for the majority of the channels I've added to the list, unless the chat of the channel is open in a tab/split?  This change has to do with #3442 or is something else ?

**Comment by Felanbird** [comment, marked as answer]
Update to the latest [nightly build](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build), it could be one of 2 issues we've fixed.
#3648 which fixed issues when the inputted name was not all lower case
#3715 which fixed an issue where Twitch would drop your entire API call if a single user had a malformed name
